### PR TITLE
[ENG-2375] [Hotfix] Tweaks and additional question for RWE Registration Schema

### PR DESCRIPTION
**Pull request by UdayVarkhedkar**
## Purpose
Make some changes to the RWE Registration Schema after it's been further reviewed

## Changes
1. Change the template name/title to "Real World Evidence Recommended Minimum Study Registration Template", old name was 'Real World Evidence in Health Outcomes Minumum Recommended Form'
2. Change q4 (Extraction date) help text to "This refers to either the version number for the database that will be used to implement the study protocol or the date that the database was created."
3. Change the q18 Question-label to "Follow-up definition"
4. Add another question after q19 for Data handling details in sheet.

(Note: qid incrementation on the schema blocks sheet is off by 1 after q6)

## QA Notes
- Verify title is "Real World Evidence Recommended Minimum Study Registration Template"
- Verify help text for q4 is in line with the schema specified in sheet
- Verify q18 question label is in line with the schema specified in sheet
- Verify q20 is in line with 'Data handling' question as specified in sheet

## Documentation
N/A

## Side Effects
N/A

## Ticket
[JIRA Ticket](https://openscience.atlassian.net/browse/ENG-2375)

### CODE REVIEW

File: website/project/metadata/real-world-evidence.json [hunk 1]
```diff
@@ -212,6 +212,14 @@
...
                "example":"If you have looked at the data to determine whether there are enough adults with sleep disordered breathing to at least satisfy requirements for your sample size, you should choose option 3 –conducted feasibility counts.",
                "required":true
+            },
+            {
+               "qid":"q20",
+               "title":"Data handling",
+               "type":"string",
+               "format":"text",
+               "help":"Clarify the description chosen for data handling.",
+               "required":true
```
Comment (brianjgeiger) [inline website/project/metadata/real-world-evidence.json:222]: I checked with Nici, and this should be optional.

**Review by brianjgeiger — CHANGES_REQUESTED** [review]
Looking good. Only one change because the sheet was ambiguous.

**Comment by brianjgeiger** [comment]
No, one more change: this should be targeting Master rather than Develop, as it is a hotfix.

**Comment by brianjgeiger** [comment]
Actually, one more thing. Could you roll in https://openscience.atlassian.net/browse/ENG-2377 to this?

**Comment by UdayVarkhedkar** [comment]
Will do - I'm going to make changes in this PR (https://github.com/CenterForOpenScience/osf.io/pull/9520) so I can start fresh with a branch based off of `master`

**Pull request closed without merging**